### ISSUE TITLE
[CLI] code devcotainer as subcommand for code CLI

### ISSUE THREAD

**Issue by ThePlenkov** [microsoft/vscode#236565]
# The Context

I'm used to develop using following set up:
- clone my project in WSL ( to have ext4 based file system )
- run VS Code with a command like code .
-  Wait when project is loaded
- Reopen in container

So if you can see to open the project in a container I always have to open first it in WSL and then in a container. So few things I'd like to mention which a bit impact the experience:

- first is time i need to spend to open project in a container, more clicks
- when opening project in WSL first then in a dev cotnainer, my VS Code history becomes a bit messy - to many projects with same name - i have to guess what is what

# Alternative approach

As we know - and that's what VS code is using inside too - there is a CLI https://github.com/devcontainers/cli which allows us to start the container with `devcontainer up`. It general - it works - however what I don't like in this case - we need to connect to the container from VS Code manually and then experience is even worse. It's not possible to rebuild the container, to re-open in WSL if needed. So VS Code experience is much better if we keep using `Reopen in container` command instead of attaching to running container which is a bit different case.

# Proposed solution

I'd like to be able to instead of `code .` something like `code devontainer up` which would start the devcontainer in the context of the current project.

So basically this command should be a shortcut for following actions:
- start VS code ( in shadow )
- Trigger Reopen in a container command
- Open VS Code immediately in a container
- If build doesn't work - it should propose to edit it in WSL ( because it was run from WSL with code devcontainer up command )

What do you think about this idea? 

Thanks!

**Comment by a-stankevich** [comment]
Related closed VSCode issue with many upvotes #111230
And related issue with with workarounds in comments https://github.com/microsoft/vscode-remote-release/issues/2133
